core/src/string_utils.rs

```diff
@@ -95,1456 +93,1458 @@ pub fn swf_string_cmp_ignore_case(a: &str, b: &str) -> std::cmp::Ordering {
         .cmp(b.chars().map(swf_char_to_lowercase))
 }
 
-static UPPERCASE_TABLE: &[(u16, u16)] = &[
```

Comment:
My concern here is that this table takes up twice the space (4 bytes per char vs 2 bytes per u16).

Reply:
That's ~1500 entries *  4 = 6KB extra for both tables.

I didn't think it a bit deal considering the current wasm binary weighs ~4.6MB (compressed ~1.5MB), but I can change it back if you want, though I can't find another way to remove the unsafe (the compiler can't elide the bounds check for us)

Reply:
I'd say let's keep the unsafe and save the bytes. (Possibly this function might change to take a `u16` eventually as we tweak our AVM String representation).

Reply:
Alright, I've reverted that commit, do you need me to rebase?

Reply:
Sure, if you could rebase away the revert commit, that'd be great. Thank you!

Reply:
Done ;)